package.json

```diff
@@ -20,7 +20,8 @@
     "test": "karma start --single-run"
   },
   "peerDependencies": {
-    "react": ">=15 || ^0.14.7"
+    "prop-types": "^15.5.8",
+    "react": ">=15"
```

Comment:
Based on the `prop-type` [readme](https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package) it looks like `prop-types` should be a dependency and not a peer dependency and it is [compatible](https://www.npmjs.com/package/prop-types#compatibility) with React 14 back to `react@0.14.9` and React 15 back to `react@15.3.0` therefore the react peer dependency could be `"react": ">=15.3.0 || ^0.14.9"`